project-koku/koku
# Identify unallocated costs in classification

## Jira Ticket

[COST-####](https://issues.redhat.com/browse/COST-####)

## Description

This change will change the `project_classification_annotation` to always be called `classification` and adds in a classification for projects that end with ` Unallocated Capacity`. 

CLASSIFICATION NAMES NEED TO BE CONFIRMED WITH UI DO NOT MERGE

## Testing

1. Checkout Branch
2. Restart Koku
3. Load test customer data with `make create-test-customer` and `make load-test-customer-data`.
4. Hit the ocp endpoint grouped by project and you should see the classifications appear
5. http://localhost:8000/api/cost-management/v1/reports/openshift/costs/?group_by%5Bproject%5D=*
6. add in a category and you should see the platform category still gets called category while the new classifications remain
7. http://localhost:8000/api/cost-management/v1/reports/openshift/costs/?group_by%5Bproject%5D=*&category=platform 

## Notes

...
